doc/apiv3-documentation.apib

```diff
@@ -1474,6 +1478,94 @@ This is an example of how a form might look like. Note that this endpoint does n
                 "message": "The resource you are about to edit was changed in the meantime."
             }
 
+# Group MyPreferences
+
+## Linked Properties
+|  Link     | Description                                              | Type          | Constraints           | Supported operations |
+|:---------:|----------------------------------------------------------| ------------- | --------------------- | -------------------- |
+| self      | This MyPreferences                                       | MyPreferences | not null              | READ                 |
+| user      | The current user that this preference belongs to         | User          | not null              | READ                 |
+
+## Local Properties
+| Property               | Description                                                | Type       | Constraints | Supported operations |
+|:----------------------:| -----------------------------------------------------------| ---------- | ----------- | -------------------- |
+| hideMail               | Hide mail address from other users                         | Boolean    |             | READ / WRITE         |
+| timeZone               | Current selected time zone                                 | String     |             | READ / WRITE         |
+| theme                  | Current selected theme                                     | String     |             | READ / WRITE         |
```

Comment:
Out of scope for your PR, but this is already asking for a form endpoint to tell me which values are allowed...